src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -124,8 +124,7 @@ protected function doSend(MessageInterface $message): SentMessage
 
         $content = $response->toArray(false);
 
-        if (0 !== (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1) {
-            $errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1;
+        if ($errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1) {
```

Comment:
Im sure this code is correct. But could we make it a bit more readable?

```php
$errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1
if (0 !== $errorCode) {
```

Reply:
@Nyholm good question :) For now do not have any suggestions how to make it more readable :(

Reply:
Yesterday, I created new method with if isset, but it is not good as understand @jderusse

Reply:
What do you think about my suggestion?

Reply:
👍 for Tobias' suggestion. That's also what #https://github.com/symfony/symfony/pull/40733#discussion_r609140232 suggested to do

Reply:
@chalasr is it not the same like here https://github.com/symfony/symfony/pull/40735? :) changed :) little bit confused now :D

Reply:
we both suggested
```
$errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1;
if (0 !== $code) {
```

you implemented
```
if ($errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1) {
```

Reply:
@jderusse sorry misunderstanding :) @Nyholm show me screen :+1: